python/tests/expressions/test_visitors.py

```diff
@@ -853,19 +868,11 @@ def test_manifest_evaluator_less_than_no_overlap():
             upper_bound=_to_byte_buffer(StringType(), "b"),
         )
     )
+    assert _ManifestEvalVisitor(string_schema, expr).eval(manifest)
 
-    assert _create_manifest_evaluator(expr).eval(manifest)
-
-
-def test_manifest_evaluator_less_than_overlap():
-    expr = BoundLessThan(
-        term=BoundReference(
-            field=NestedField(field_id=1, name="foo", field_type=StringType(), required=False),
-            accessor=Accessor(position=0, inner=None),
-        ),
-        literal=StringLiteral("a"),
-    )
 
+def test_manifest_evaluator_less_than_overlap(string_schema: Schema):
```

Comment:
Isn't this the case without overlap? Values < "a" doesn't overlap values from "a" to "b".

Reply:
`{a} ∩ {a, b} = {a}`